tree/tree/src/TLeafElement.cxx

```diff
@@ -38,7 +38,7 @@ TLeafElement::TLeafElement(): TLeaf()
 
 TLeafElement::TLeafElement(TBranch *parent, const char *name, Int_t id, Int_t type)
    : TLeaf(parent, name,name),
-     fUnknownDataSize(false)
+     fLenType(0)
```

Comment:
This commit should be a fixup for the one that adds the data member.

Reply:
Done.